src/MiNET/MiNET/Net/MCPE Protocol.cs

```diff
@@ -3646,6 +3646,7 @@ public partial class McpeBlockPickRequest : Package<McpeBlockPickRequest>
 		public int x; // = null;
 		public int y; // = null;
 		public int z; // = null;
+		public bool useUserData;
```

Comment:
When you update packets, you do this in the XML. Then you select the menu in Visual Studio -> "Build | Transform all T4 Templates" and it will generate the code.

Reply:
I've commited a fix to the packet itslef. So just need your implementation of the handler itself.

Reply:
Yeah, that's fair.
Wasn't entirely sure where they were updated.